training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -1,3 +1,9 @@
 azure-core=={{latest-pypi-version}}
 azure-ai-ml=={{latest-pypi-version}}
 azure-identity=={{latest-pypi-version}}
+protobuf==3.20.0
+sentencepiece==0.1.97
+accelerate==0.17.1
+datasets==2.3.2
+diffusers==0.14.0
+https://automlcesdkdataresources.blob.core.windows.net/finetuning-models/eval_metrics_whls/17_03_latest_eval_wheel/azureml_evaluate_mlflow-0.1.0.86769376-py3-none-any.whl
```

Comment:
I thought we install from private feeds/blobs for private preview. For public those should be on pypi

Reply:
updated to use PyPi